### PR TITLE
Added troubleshooting folder

### PR DESCRIPTION
**Pull request by Epiphic**
This folder is for users to reference to or add to when an issue is found.

Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.
- [x] **I have mentioned this change in the [changelog](https://github.com/CCExtractor/ccextractor/blob/master/docs/CHANGES.TXT).**

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [x] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---

{pull request content here}

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
Can you try to make it an .md? That's more universally accessible on GitHub compared to PDF 👍

File: troubleshooting/Solving v141_xp build issue.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### **Guide to solving v141_xp build issue in Visual Studio 2015+**
+
+
+With only the default build tools installed you might receive an error that looks like this when trying to build:
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/v141_xp-img1)
```
Comment (MatejMecka) [inline troubleshooting/Solving v141_xp build issue.md]: Can you write a description of the image instead of saying alt text on all of them?

**Comment by rogersachan** [comment]
Please see https://github.com/CCExtractor/ccextractor/wiki/Building-CCExtractor-with-Visual-Studio and https://github.com/CCExtractor/ccextractor/wiki/Solving-build-issue-with-the-v141_xp-toolchain which I have done for GCI. I have also embedded the images in these using the proper commonmark formatting.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -40,7 +40,12 @@ configure_file (
   "${PROJECT_SOURCE_DIR}/lib_ccx/compile_info_real.h"
   )
 
-add_definitions(-DVERSION_FILE_PRESENT -DFT2_BUILD_LIBRARY -DGPAC_DISABLE_VTT -DGPAC_DISABLE_OD_DUMP)
+#Check the operating system and add the GPAC bits if its 64bit
+IF (CMAKE_SIZEOF_VOID_P EQUAL 8)
+    add_definitions(-DVERSION_FILE_PRESENT -DFT2_BUILD_LIBRARY -DGPAC_DISABLE_VTT -DGPAC_DISABLE_OD_DUMP -DGPAC_64_BITS)
+ELSE(CMAKE_SIZEOF_VOID_P EQUAL 8)
+	add_definitions(-DVERSION_FILE_PRESENT -DFT2_BUILD_LIBRARY -DGPAC_DISABLE_VTT -DGPAC_DISABLE_OD_DUMP)
+ENDIF(CMAKE_SIZEOF_VOID_P EQUAL 8)
```
Comment (anshul1912) [inline src/CMakeLists.txt]: Already present in code
```
add_definitions(-DVERSION_FILE_PRESENT -DFT2_BUILD_LIBRARY -DGPAC_DISABLE_VTT -DGPAC_DISABLE_OD_DUMP)

if(CMAKE_SIZEOF_VOID_P EQUAL 8)
  add_definitions(-DGPAC_64_BITS)
endif()
```

File: troubleshooting/Guide to building CCExtractor windows.exe in Visual Studio.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img2)
+
+The project will open, you can see the tree on the right. Right click the ccextractor project and click build.
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img3)
+
+It should take a few moments, then say build successful:
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img4)
```
Comment (anshul1912) [inline troubleshooting/Guide to building CCExtractor windows.exe in Visual Studio.md]: we already have "https://github.com/CCExtractor/ccextractor/wiki/Installation#windows" I dont think we need this info again

File: troubleshooting/Solving v141_xp build issue.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/v141_xp-img2)
+
+I have Visual Studio Community 2017, but whatever you have installed should be on the top left. Click modify. Once the new screen opens, there will be options on the right side, click Desktop development with C++.
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/v141_xp-img3)
+
+Under optional, select the the two highlighted tools in the picture above. In the bottom right corner it will show the amount of space required for the addition, ensure you have enough space on your disk, then click Modify. If Visual Studio doesn't open up after modifying, open the ccextractor.sln again. Right click the project and click build, and this time it should be successful!
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/v141_xp-img4)
```
Comment (anshul1912) [inline troubleshooting/Solving v141_xp build issue.md]: Instead of new folder, I would suggest to put troubleshooting information to go in wiki.

**Pull request closed without merging**